rpm/iptables.spec

```diff
@@ -1,11 +1,12 @@
 Name:       iptables
 Summary:    Tools for managing Linux kernel packet filtering capabilities
-Version:    1.8.10
+Version:    1.8.11
 Release:    1
 License:    GPLv2
 URL:        http://www.netfilter.org/projects/iptables
```

Comment:
You could change this to point to this repo instread of upstream.

Reply:
Changed in a192cfc6000334aa643f0db764fead6dff5e7bbc